src/utils.js

```diff
@@ -199,3 +199,12 @@ export function humanDate(since: string | number): string {
     second: "2-digit",
   }) + " UTC";
 }
+
+export function buildAttachmentUrl(record: RecordData, capabilities: Capabilities): ?string {
+  if (!record.attachment || typeof capabilities.attachments !== "object") {
+    return;
+  }
+  const {base_url} = capabilities.attachments;
+  const {location} = record.attachment;
+  return location.startsWith("http") ? location : base_url + location;
```

Comment:
> What about return location.startsWith("base_url")

You mean testing for the variable, not this string, right?

> Can we do something clever so that we make sure that location starts with a / and base_url doesn't end with one?

I know you don't, but if you ask me, this should be handled by the attachment plugin (also the plugin could always make the `location` field exposing a full url based on the `base_url` config flag methinks).

Anyway, yes I'll add supplementary checks for this.